apps/projects/app/components/Shared/BountyContextMenu.js

```diff
@@ -53,7 +53,13 @@ const BountyContextMenu = ({ issue }) => {
           <Item onClick={() => reviewWork(issue)}>
             <IconView color={`${theme.surfaceContent}`} />
             <ActionLabel>
-              Review Work
+              View work
+            </ActionLabel>
+          </Item>
+          <Item onClick={() => reviewApplication(issue)}>
+            <IconView color={`${theme.surfaceIcon}`} />
+            <ActionLabel>
+              View aplications ({issue.requestsData.length})
```

Comment:
typo: "aplications" is missing a P

Reply:
@stellarmagnet it adds complexity, but should we say "View application" if there's only one, and "View applications (N)" if there's more than one, where N is the count of them?

Reply:
Fixed the typo, thank you!